lib/nc-time-axis/__init__.py

```diff
@@ -0,0 +1,213 @@
+# (C) British Crown Copyright 2016, Met Office
+#
+# This file is part of nc-time-axis.
+#
+# nc-time-axis is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# nc-time-axis is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with nc-time-axis. If not, see <http://www.gnu.org/licenses/>.
+"""
+Support for netcdftime axis in matplotlib.
+
+"""
+
+from collections import namedtuple
+import datetime
+
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+import matplotlib.transforms as mtransforms
+import matplotlib.units as munits
+import netcdftime
+import numpy as np
+
+
+class NetCDFtimeConverter(mdates.DateConverter):
+    standard_unit = 'days since 2000-01-01'
```

Comment:
Strange choice of epoch. Although this can be anything, it tends to be `1970-01-01`